.cirrus/tasks.yml

```diff
@@ -153,7 +155,7 @@ private_scan_task:
     - java -jar /usr/local/bin/wss-unified-agent.jar -c .cirrus/wss-unified-agent.config -apiKey $MEND_API_KEY -product ${WS_PRODUCTNAME} -project ${STAGING_IMAGE_NAME}:${tag} -wss.url ${WS_WSS_URL} -docker.scanImages true
   depends_on:
   - multi_arch_build_9_x
-  - multi_arch_build_10_x
+  - multi_arch_build_sonarqube_server_task
```

Comment:
`_task` should not be present in `depends_on` entries; also the task `multi_arch_build_sonarqube_server` does not exist in this file ?

Reply:
It does exist: line 118, as an alias. :)